.gitignore

```diff
@@ -23,3 +23,5 @@ measure_bw.sh
 *.blob
 *.blob.sz3
 *.ipynb
+
+a.py
```

Comment:
🗑️ ♻️

Reply:
whats the problem with that?

Reply:
Its just random? I thought mainly, that its a leftover and slipped in here by accident

Reply:
well I mean its just a random script I used for testing, but I guess I can just put it into my local ignore settings